packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/pages/search/[[...alias]].jsx

```diff
@@ -71,12 +91,34 @@ export async function getServerSideProps(context) {
 			anon: true,
 		});
 
+		const [searchTerm] = alias ? alias : [null];
+		const searchResults = searchTerm
+			? (
+					await getDrupalSearchResults({
+						apiUrl: process.env.BACKEND_URL,
+						locale: locale,
+						query: searchTerm,
+						response: res,
+					})
+			  ).data.map((value) => {
+					// restructure response to match expected article object structure
+					return value.attributes;
+			  })
+			: null;
+
+		// Do not send error unless search term is present so /search remains a valid path
+		if (!searchResults && searchTerm) {
+			errorMessage = true;
+		}
```

Comment:
I don't really understand this. It is possible to get 0 search results but there is no error.

Reply:
If I query 'test' I get 0 search results and no error, but also no message as to the search having been done or not.